backend/src/emails/service.py

```diff
@@ -0,0 +1,28 @@
+from fastapi_mail import FastMail, MessageSchema, MessageType
+from starlette.responses import JSONResponse
+
+from src.core.config import settings
+
+
+def generate_email(subject: str, recipients: list, template_body: dict,
+                   subtype: MessageType = MessageType.html) -> MessageSchema:
+    message = MessageSchema(
+        subject=subject,
+        recipients=recipients,
+        template_body=template_body,
+        subtype=subtype)
+    return message
+
+
+async def send_email(message: MessageSchema, template_name: str):
+    fm = FastMail(settings.MAIL_CONNECTION_CONF)
+    await fm.send_message(message, template_name=template_name)
+    return JSONResponse(status_code=200, content={"message": "email has been sent"})
```

Comment:
don't return json response in a service function. it's not a part of the API. do it in the endpoint function instead